src/Apps/Order/Routes/__tests__/Status.test.tsx

```diff
@@ -1,32 +1,56 @@
 import { Message } from "@artsy/palette"
-import { mount, render } from "enzyme"
+import { render } from "enzyme"
 import React from "react"
-import { HeadProvider } from "react-head"
-import { Status_order } from "../../../../__generated__/Status_order.graphql"
+import { graphql } from "react-relay"
 import {
+  mockResolver,
   OrderWithShippingDetails,
   PickupOrder,
 } from "../../../../Apps/__test__/Fixtures/Order"
-import { StatusRoute } from "../Status"
+import { MockBoot, renderRelayTree, UntilCallback } from "../../../../DevTools"
```

Comment:
Once TypeScript support is fixed in our tests these auto imports will be absolute